docs/API.md

```diff
@@ -1541,22 +1542,22 @@ if err != nil {
 ### GetBucketLifecycle(bucketName) (lifecycle string, error)
 Get lifecycle on a bucket or a prefix.
 
-__Parameters__
+__参数__
 
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
 |`bucketName`  | _string_  |Name of the bucket   |
 
-__Return Values__
+__返回类型__
 
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
 |`lifecycle`  | _string_ |Lifecycle returned from the server |
 |`err` | _error_  |Standard Error  |
 
-__Example__
+__示例__
```

Comment:
These should be removed. This is the english version of the document.

Reply:
Thank you very much, I'll modify it.